src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    @include square($star-size);
+    background-image: url("/images/star-active.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(n+5) {
```

Comment:
will your code work if the number of active stars need to be change? you need reuse all your star code, it need to work with options stars--4, stars--3 and changable

Reply:
Agree with @TheVovchik, fix it